src/utils/gitUtils.ts

```diff
@@ -75,10 +76,24 @@ export async function verifyGitWorkspaceForCreation(context: IActionContext, git
         gitWorkspaceState.repo = newRepo;
     } else if (!!gitWorkspaceState.remoteRepo && !gitWorkspaceState.hasAdminAccess) {
         context.telemetry.properties.cancelStep = 'adminAccess';
+        const adminAccess: string = localize('adminAccess', 'Admin access to the GitHub repository "{0}" is required. Would you like to create a fork?', gitWorkspaceState.remoteRepo.name);
+        const createForkItem: MessageItem = { title: localize('createFork', 'Create Fork') };
+        await ext.ui.showWarningMessage(adminAccess, { modal: true }, createForkItem);
 
-        const adminAccess: string = localize('adminAccess', 'Admin access to the GitHub repository is required.  Use a repo with admin access or create a fork.');
-        await ext.ui.showWarningMessage(adminAccess, { modal: true });
+        const repoUrl: string = (await createFork(context, gitWorkspaceState.remoteRepo)).data.html_url;
 
+        context.telemetry.properties.cancelStep = 'cloneFork';
+        const forkSuccess: string = localize('forkSuccess', 'Successfully forked "{0}". Would you like to clone your new repository?', gitWorkspaceState.remoteRepo.name);
+        const clone: MessageItem = { title: localize('clone', 'Clone Repo') };
+        ext.outputChannel.appendLog(forkSuccess);
```

Comment:
Kind of a nit, but.... 

nit: Could you remove the question "Would you like to clone your new repository?" from the appendLog string?  It doesn't make any sense in the context of the output channel 😬 

I'd understand if you didn't.

If it doesn't bother you, then don't worry about it.

...

👀

Reply:
😄 No you're right, that looks goofy. Consider it gone 😤